third_party/CMakeLists.txt

```diff
@@ -146,6 +146,63 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
   set(VULKAN_LOADER_BUILD_FIXES
     -Wno-unknown-warning-option
   )
+
+  set(SWIFTSHADER_BUILD_FIXES
```

Comment:
oh my

Reply:
Also you need some more options for GNU.